doc/radosgw/notifications.rst

```diff
@@ -144,6 +148,20 @@ Request parameters:
   - "none": message is considered "delivered" if sent to broker
   - "broker": message is considered "delivered" if acked by broker (default)
 
+- AWS Endpoint
+
+ - push-endpoint: ``aws://sns.amazonaws.com`` for AWS SNS and ``aws://lambda.amazonaws.com`` for AWS lambda.
+ - use-ssl: if "true" then secure connection will be used for connecting with the broker
+ - AccessKey: Access Key for access to AWS services
+ - SecretKey: Access secret for access to AWS services
+ - AwsArn: ARN of the resource to push notification to.
+ - verify-ssl: indicate whether the server certificate is validated by the client or not ("false" by default)
+ - aws-ack-level: no end2end acking is required, as messages may persist in the broker before delivered into their final destination. Two ack methods exist:
```

Comment:
the first sentence only apply to amqp - please remove.
however, we should probably add these clarifications:
* in case of SNS message is considered delivered when pushed into SNS and does not have to be notified to the end recipients
* in case of lambda, we ignore the success/fail of the execution of the function, and consider the message delivered when the function is successfully triggered